mono/utils/mono-threads.h

```diff
@@ -627,4 +627,10 @@ gboolean mono_threads_wait_pending_operations (void);
 void mono_threads_begin_global_suspend (void);
 void mono_threads_end_global_suspend (void);
 
+static inline gboolean
```

Comment:
Don't mark static header functions as inline. It confuses the win32 linker and any compiler under O2 will inline it anyways.